components/Carbon.js

```diff
@@ -178,7 +184,7 @@ class Carbon extends React.Component {
             }
 
             #container :global(.CodeMirror-gutters) {
-              background-color: unset;
+              background-color: unset !important;
```

Comment:
some themes have `!important` in their background thereby rendering this useless.